patches/musescore-webengine.diff

```diff
@@ -0,0 +1,39 @@
+diff --git a/main/CMakeLists.txt b/main/CMakeLists.txt
+index 6c489e86c..66bddcf01 100644
+--- a/main/CMakeLists.txt
++++ b/main/CMakeLists.txt
+@@ -218,20 +218,20 @@ else (MINGW)
+ 
+    if ( NOT MSVC )
+       ## install qwebengine core
+-      if (NOT APPLE AND USE_WEBENGINE)
+-         install(PROGRAMS
+-            ${QT_INSTALL_LIBEXECS}/QtWebEngineProcess
+-            DESTINATION bin
+-            )
+-         install(DIRECTORY
+-            ${QT_INSTALL_DATA}/resources
+-            DESTINATION lib/qt5
+-            )
+-         install(DIRECTORY
+-            ${QT_INSTALL_TRANSLATIONS}/qtwebengine_locales
+-            DESTINATION lib/qt5/translations
+-            )
+-      endif(NOT APPLE AND USE_WEBENGINE)
++##      if (NOT APPLE AND USE_WEBENGINE)
```

Comment:
Just a style nitpick, why not simply remove this part?

Reply:
no specific reason.